src/controller/CommissioningDelegate.h

```diff
@@ -51,6 +54,9 @@ enum CommissioningStage : uint8_t
     kSendAttestationRequest,     ///< Send AttestationRequest (0x3E:0) command to the device
     kAttestationVerification,    ///< Verify AttestationResponse (0x3E:1) validity
     kAttestationRevocationCheck, ///< Verify Revocation Status of device's DAC chain
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+    kJCMTrustVerification,       ///< Perform JCM trust verification steps
+#endif // CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
```

Comment:
This was covered in the review comments on https://github.com/project-chip/connectedhomeip/pull/38749: it's pretty undesirable for the numbering of these stages to be unstable depending on whether JSM is enabled.  Either put the ifdef bit at the end of the enum, or don't ifdef it at all, please.